src/status.h

```diff
@@ -130,6 +130,9 @@ typedef enum
 
     NATS_MISSED_HEARTBEAT,              ///< For JetStream subscriptions, it means that the library detected that server heartbeats have been missed.
 
+    NATS_LIMIT_REACHED,                 ///< Attempt to receive messages than allowed by the byte limit, for
```

Comment:
Note that adding new values here requires that you bump the `NATS_VERSION_REQUIRED_NUMBER` in the main `CMakeLists.txt`. See `nats_CheckCompatibilityImpl()` for more information.

Reply:
@kozlovic We are already set for 3.9 (next) in `main`, https://github.com/nats-io/nats.c/blob/4f19da67f801c61560c652bb84680ba021e042d7/CMakeLists.txt#L254-L259

Does it need to change?

Reply:
No, if it is already done.